app/controllers/auth/passwords_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Auth::PasswordsController < Devise::PasswordsController
   include Helpers::Email
+  before_action :set_minimum_password_length, only: %i[edit create]
```

Comment:
this shows the password hint text that was previously nil - added to this ticket as it's a small improvement, see ticket

Reply:
Yep, was just trying to avoid repetition but I've ended up just writing more code!